src/reference/informative/informC08_interpretation_of_mathematics2.rst

```diff
@@ -0,0 +1,217 @@
+          <connection component_1="FirstGame" component_2="Tom" >
+            <map_variables variable_1="toms_runs" variable_2="runs" />
+          </connection>
+          <!-- Since component "DickTheDog" is too distant in the encapsulation structure to be
+              visible to the "FirstGame" component, any runs scored by the dog must be passed
+              through Harry's component. -->
+          <connection component_1="FirstGame" component_2="Harry" >
+            <map_variables variable_1="harrys_runs" variable_2="runs" />
+            <map_variables variable_1="dicks_runs" variable_2="dicks_runs" />
+          </connection>
+
+          <encapsulation>
+            <component_ref component="FirstGame" >
+              <component_ref component="Tom" />
+              <component_ref component="Harry" />
+            </component_ref>
+          </encapsulation>
+        </model>
+
+        <!-- In file: Neighbours.cellml -->
+        <model name="HarrysHouse" >
+
+          <component name="DickTheDog">
+            <variable name="runs" units="dimensionless" />
+            <math>
+              <!-- This statement sets Dick's score to 0 for all time.  
+                  He's a dog.  He can't use a cricket bat. -->
+              <apply><eq/>
+                <ci>runs</ci>
+                <cn cellml:units="dimensionless">0</cn>
+              </apply>
+            </math>
+          </component>
+
+          <component name="Harry">
+            <variable name="time" units="minute" />
+            <variable name="runs" units="dimensionless" />
+            <math>
+              <!-- This statement represents DickTheDog running away with the ball, 
+                  enabling Harry to score an ever-increasing number of runs. -->
+              <apply><eq/>
+                <ci>runs</ci>
+                <apply><times/>
+                  <cn cellml:units="per_minute">10</cn>
+                  <ci>time</ci>
+              </apply>
+            </math>
+          </component>
+
+          <!-- A connection is established between Harry and Dick to enable sharing of their run tally. -->
+          <connection component_1="Harry" component_2="DickTheDog" >
+            <map_variables variable_1="dicks_runs" variable_2="runs" />
+          </connection>
+          ...
+        </model>
+
+
+    .. container:: heading3
+
+      Understanding how and when the mathematics *doesn't* work
+
+    It's possible to write valid CellML that does not represent valid mathematics.
+    You can think of this like correctly spelling a set of words which together do not form a meaningful sentence.
+    Some examples of valid versus valid-but-nonsense :code:`math` elements' contents are shown below.
+    It's anticipated that good software implementations of CellML will check for these kinds of situations and inform the user.
+
+    Simple over-definition is valid, but will result in undefined behaviour:
```

Comment:
Maybe just stick to saying "Simple over-definition:" ?
The CellML spec doesn't really define behaviour of tools that use CellML docs

Reply:
Maybe just say that it's valid CellML, but hard to see how this could be interpreted meaningfully?

Reply:
1730324